tests/test_distributions/test_family.py

```diff
@@ -2,7 +2,7 @@
 import pytest
 
 from causalprog.distribution.base import SampleTranslator
-from causalprog.distribution.family import DistributionFamily
+from causalprog.distribution.normal import DistributionFamily
```

Comment:
```suggestion
from causalprog.distribution.family import DistributionFamily
```

Not sure why it's now importing from a different module.

Reply:
I think it's nicer to import from family as that's where this is defined?